mycore-solr/src/main/resources/xsl/solr-xeditor-templates.xsl

```diff
@@ -192,10 +194,23 @@
   </xsl:template>
 
   <xsl:template match="mcrsolr:fieldsHelp">
+    <xsl:apply-templates select="." mode="with-core">
+      <xsl:with-param name="core" select="$MCR.Solr.Core.main.Name" />
+      <xsl:with-param name="type" select="'main'" />
+    </xsl:apply-templates>
+    <xsl:apply-templates select="." mode="with-core">
+      <xsl:with-param name="core" select="$MCR.Solr.Core.classification.Name" />
+      <xsl:with-param name="type" select="'classification'" />
+    </xsl:apply-templates>
+  </xsl:template>
+  <xsl:template match="mcrsolr:fieldsHelp" mode="with-core">
+    <xsl:param name="core" />
+    <xsl:param name="type" />
     <div class="table-responsive">
       <table class="table table-striped table-hover table-condensed">
-        <xsl:variable name="url" select="concat($MCR.Solr.ServerURL, '/admin/luke')" />
+        <xsl:variable name="url" select="concat($MCR.Solr.ServerURL, 'solr/', $core, '/admin/luke?wt=xml')" />
```

Comment:
It is possible to specify a ServerURL for each Core, MCR.Solr.ServerURL is only the fallback if no URL for core is specified

Reply:
I'll take a look on this